projects/1.16/assets/extra-armor/extraarmor/lang/zh_cn.json

```
@@ -1 +1,115 @@
+  "item.extraarmor.bone_leggings": "骨头护腿",
+  "item.extraarmor.bone_boots": "骨头靴子",
+  "item.extraarmor.molten_helmet": "熔融头盔",
+  "item.extraarmor.molten_chestplate": "熔融胸甲",
+  "item.extraarmor.molten_leggings": "熔融护腿",
+  "item.extraarmor.molten_boots": "熔融靴子",
+  "item.extraarmor.pumpkin_helmet": "南瓜头盔",
+  "item.extraarmor.pumpkin_chestplate": "南瓜胸甲",
+  "item.extraarmor.pumpkin_leggings": "南瓜护腿",
+  "item.extraarmor.pumpkin_boots": "南瓜靴子",
+  "item.extraarmor.phantom_helmet": "幻影头盔",
+  "item.extraarmor.phantom_chestplate": "幻影胸甲",
+  "item.extraarmor.phantom_leggings": "幻影护腿",
+  "item.extraarmor.phantom_boots": "幻影靴子",
+  "item.extraarmor.ghost_helmet": "幽灵头盔",
+  "item.extraarmor.ghost_chestplate": "幽灵胸甲",
+  "item.extraarmor.ghost_leggings": "幽灵护腿",
+  "item.extraarmor.ghost_boots": "幽灵靴子",
+  "item.extraarmor.crying_obsidian_helmet": "哭泣的黑曜石头盔",
+  "item.extraarmor.crying_obsidian_chestplate": "哭泣的黑曜石胸甲",
+  "item.extraarmor.crying_obsidian_leggings": "哭泣的黑曜石护腿",
+  "item.extraarmor.crying_obsidian_boots": "哭泣的黑曜石靴子",
+  "item.extraarmor.ice_helmet": "冰头盔",
+  "item.extraarmor.ice_chestplate": "冰胸甲",
+  "item.extraarmor.ice_leggings": "冰护腿",
+  "item.extraarmor.ice_boots": "冰靴子",
+  "item.extraarmor.snow_helmet": "雪头盔",
+  "item.extraarmor.snow_chestplate": "雪胸甲",
+  "item.extraarmor.snow_leggings": "雪护腿",
+  "item.extraarmor.snow_boots": "雪靴子",
+  "item.extraarmor.glowstone_helmet": "荧石头盔",
+  "item.extraarmor.glowstone_chestplate": "荧石胸甲",
+  "item.extraarmor.glowstone_leggings": "荧石护腿",
+  "item.extraarmor.glowstone_boots": "荧石靴子",
+  "item.extraarmor.prismarine_shard_helmet": "海晶碎片头盔",
+  "item.extraarmor.prismarine_shard_chestplate": "海晶碎片胸甲",
+  "item.extraarmor.prismarine_shard_leggings": "海晶碎片护腿",
+  "item.extraarmor.prismarine_shard_boots": "海晶碎片靴子",
+  "item.extraarmor.prismarine_crystal_helmet": "海晶砂粒头盔",
+  "item.extraarmor.prismarine_crystal_chestplate": "海晶砂粒胸甲",
+  "item.extraarmor.prismarine_crystal_leggings": "海晶砂粒护腿",
+  "item.extraarmor.prismarine_crystal_boots": "海晶砂粒靴子",
+  "item.extraarmor.honeycomb_helmet": "蜜脾头盔",
+  "item.extraarmor.honeycomb_chestplate": "蜜脾胸甲",
+  "item.extraarmor.honeycomb_leggings": "蜜脾护腿",
+  "item.extraarmor.honeycomb_boots": "蜜脾靴子",
+  "item.extraarmor.emerald_helmet": "绿宝石头盔",
+  "item.extraarmor.emerald_chestplate": "绿宝石胸甲",
+  "item.extraarmor.emerald_leggings": "绿宝石护腿",
+  "item.extraarmor.emerald_boots": "绿宝石靴子",
+  "item.extraarmor.ender_helmet": "末影头盔",
+  "item.extraarmor.ender_chestplate": "末影胸甲",
+  "item.extraarmor.ender_leggings": "末影护腿",
+  "item.extraarmor.ender_boots": "末影靴子",
+  "item.extraarmor.feather_helmet": "羽毛头盔",
+  "item.extraarmor.feather_chestplate": "羽毛胸甲",
+  "item.extraarmor.feather_leggings": "羽毛护腿",
+  "item.extraarmor.feather_boots": "羽毛靴子",
+  "item.extraarmor.slime_helmet": "史莱姆头盔",
+  "item.extraarmor.slime_chestplate": "史莱姆胸甲",
+  "item.extraarmor.slime_leggings": "史莱姆护腿",
+  "item.extraarmor.slime_boots": "史莱姆靴子",
```

```suggestion
  "item.extraarmor.slime_helmet": "黏液头盔",
  "item.extraarmor.slime_chestplate": "黏液胸甲",
  "item.extraarmor.slime_leggings": "黏液护腿",
  "item.extraarmor.slime_boots": "黏液靴子",
```
参考匠魂3，slime作装备用时应为“黏液”。

---

好奇匠魂3采用黏液的原因是什么

---

> 好奇匠魂3采用黏液的原因是什么

星落辰表示：我们将生物性质的Slime/Slimy翻译成“史莱姆xx”，将物品性质的Slime/Slimy翻译成“黏液xx”，从而与原版保持一致。